cadnano/gui/views/customqgraphicsview.py

```diff
@@ -328,20 +332,18 @@ def keyPressEvent(self, event):
         elif event.key() == Qt.Key_Minus:
             self.zoomIn(0.03)
         else:
-            return QGraphicsView.keyPressEvent(self, event)
-        # end else
+            if hasattr(self.scene_root_item, KEY_PRESS_EVENT):
+                getattr(self.scene_root_item, KEY_PRESS_EVENT)(event)
```

Comment:
This change would appear to be necessary to `customqgraphicsview.py`; prior to this change, a `keyPressEvent` when `event.key()` is `Qt.Key_Escape` would end up in
```
QGraphicsView.keyPressEvent(self, event)
```

The `QGraphicsView.keyPressEvent` simply `pass`es, so nothing would happen and the `KeyPressEvent` would be ignored and not passed on to `scene_root_item`

This change effectively does something with the event rather than discarding it

Reply:
Great, thanks for looking into this.